docs/apps/goaccess.md

```diff
@@ -0,0 +1,6 @@
+# GoAccess
+
+[GoAccess](https://goaccess.io/) is an Alpine linux container which builds GoAccess including GeoIP. It reverse proxies the GoAccess HTML files and websockets through nginx, allowing GoAccess content to be viewed without any other setup. GoAccess itself is an open source real-time web log analyzer and interactive viewer that runs in a terminal in *nix systems or through your browser. It provides fast and valuable HTTP statistics for system administrators that require a visual server report on the fly.
```

Comment:
The `*` in `*nix` seems to be making GH think that the text after it should be italic. The markdown itself seems fine, but can we replace it with `linux` anyway? (unlikely that we'll be making DS work on anything other than linux in the near future).